CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
    igraph's own RNG if needed instead of relying on LAPACK or ARPACK to do so.
    This makes sure that the results obtained from these functions remain the
    same if igraph's RNG is seeded with the same value.
+ - `igraph_distances_johnson()` now takes a mode parameter to determine in which direction paths should be followed.
```

Comment:
Ah yes, one more thing:

This line shouldn't be added here. Instead, the changelog should be separated to two parts, one for 0.10.x and one for 0.11. This should be added to the latter.

We need to come up with different headers for these. I would actually suggest just writing `[0.10.2] - Unreleased` and `[0.11.0] - Unreleased`, i.e. use "Unreleased" in place of the date instead of in place of the version. @ntamas Are you fine with this notation?

The current header should be updated on `master` first.

Reply:
Having two headers is OK, but note that the bracketed parts are actually links to be resolved by the Markdown renderer -- there's a list at the bottom of the changelog that tells which Github URLs the bracketed stuff refers to. We need to add [0.10.2] and [0.11.0] there, one referring to `master`, the other to `develop`. But if we do that, we could even call the branches by names and not by upcoming version numbers.

Reply:
@ntamas Can you just update the `CHANGELOG` on `master` with your preferred phrasing? Then we can adapt this PR to it.

Reply:
Added the `master` section on the branch, and I'll add the `develop` section here